.github/workflows/draw-zmk.yml

```
@@ -30,10 +30,15 @@ on:
         required: false
         type: string
       commit_message:
-        description: "Commit message for updated images"
-        default: "[Skip CI] keymap-drawer render"
+        description: "Commit message for updated images. Ignored if `amend_commit` is `true`."
+        default: "keymap-drawer render"
         required: false
         type: string
+      amend_commit:
+        description: "**WARNING**: re-writes history! Make sure you understand the implications before proceeding.\r\nWhether to amend the last commit instead of creating a new one."
```

```suggestion
        description: "Whether to amend the last commit instead of creating a new one. Make sure you understand the implications of rewriting the branch history if you use this option!"
```
I think a little shorter would be good here, given the detailed warning in the readme